helsinki_notification/models.py

```diff
@@ -96,6 +97,16 @@ def _localized_title(self):
     def __str__(self):
         return self._localized_title
 
+    def save(self, *args, **kwargs):
+        self.clean()
+        return super().save(*args, **kwargs)
+
+    def clean(self):
+        if self.validity_period_start > self.validity_period_end:
```

Comment:
Would it be reasonable to have gte?

```suggestion
        if self.validity_period_start >= self.validity_period_end:
```

would be more in line with the validation error message and also I imagine having zero duration validity_period makes no sense...?

Reply:
Yeah, sounds reasonable. It would make more sense. Will change it.

Reply:
Changed to gte. Also changed other code related to validity periods.